code/src/components/ScannedInfo.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { data } from '../reducers/data'
+import { Card } from '../lib/Card'
+import styled from 'styled-components/macro'
+
+
+export const ScannedInfo = () => {
+  const scannedItem = useSelector((state) => state.data.item)
+  
+  return (
+  <>
+      {scannedItem.product && scannedItem.status === 1 && (
```

Comment:
Good that you're checking this! Did you find a status code that you can use when no product is found as well?